custom/ewsghana/alerts/ongoing_non_reporting.py

```diff
@@ -12,7 +12,8 @@ class OnGoingNonReporting(WeeklyAlert):
     message = ONGOING_NON_REPORTING
 
     def get_sql_locations(self):
-        return SQLLocation.objects.filter(domain=self.domain, location_type__name='district')
+        return SQLLocation.objects.filter(domain=self.domain, location_type__name='district')\
```

Comment:
you can also use `SQLLocation.active_objects` for this. might be cleaner. https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/locations/models.py#L218